packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testUS07RedoButtonRedoesChanges.st

```diff
@@ -1,12 +1,10 @@
 tests
 testUS07RedoButtonRedoesChanges
 
-	| redoButton text |
-	redoButton := self editorView findByLabel: 'Redo'.
+	| text |
```

Comment:
Better use the instance method `text` instead of creating a new text (I'm sure that is a historic test, lets just change that now)